modules/admin-top-bar/assets/js/components/connection-button/connection-button.js

```diff
@@ -0,0 +1,23 @@
+import BarButton from '../bar-button/bar-button';
+
+export default function ConnectionButton( props ) {
+	const isUserConnected = elementorCommonConfig.connect.is_user_connected;
```

Comment:
Use the elementorConnect plugin like on 
https://github.com/matipojo/elementor/blob/e86b957a2ebdf75a4c4c74c18f8e6adf33a3df99/assets/dev/js/editor/components/template-library/views/parts/connect.js#L16

Reply:
Done.